pkg/reconciler/mongodb/resources/receive_adapter.go

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
same with this one. Just remove them from this PR, but keep them somewhere for a follow up